package.json

```diff
@@ -20,18 +20,18 @@
   "devDependencies": {
     "bower": "1.7.9",
     "coveralls": "2.11.9",
-    "grunt": "1.0.1",
+    "grunt": "^1.0.3",
```

Comment:
Note that this update breaks the travis build on node 0.12. I'm totally fine with removing the node 0.12 CI, you can remove it from the travis configuration file to make the CI pass.